src/checkstyle/checks/design/InterfaceCheck.hx

```diff
@@ -4,15 +4,17 @@ import checkstyle.token.TokenTree;
 import haxe.macro.Expr;
 import checkstyle.LintMessage.SeverityLevel;
 
-@name("InterfaceIsType")
-@desc("Checks for interfaces that does not contain any methods but only constants")
-class InterfaceIsTypeCheck extends Check {
+@name("Interface")
+@desc("Checks and enforces interface style (allow properties and methods or just methods")
+class InterfaceCheck extends Check {
 
 	public var allowMarkerInterfaces:Bool;
+	public var allowProperties:Bool;
```

Comment:
Because detecting `get/set` access identifier using token tree is not straight forward.

They are not keywords and just like any other `Const` and it's not safe to do string comparison.